test/codeu/chat/server/DerbyStoreTest.java

```diff
@@ -0,0 +1,190 @@
+package codeu.chat.server;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+import org.junit.Before;
+
+import codeu.chat.DerbyStore;
+import codeu.chat.common.User;
+import codeu.chat.util.Uuid;
+import codeu.chat.common.BasicController;
+import codeu.chat.common.Conversation;
+import codeu.chat.common.Message;
+
+public class DerbyStoreTest {
+	
+	private DerbyStore ds;
+	private Model model;
+	private BasicController controller;
+		
+	@Before
+	public void doBefore() {
+		ds = new DerbyStore();
+		model = new Model();
+	    controller = new Controller(Uuid.NULL, model);
```

Comment:
nit: indentation. If you're using Eclipse, autoformatting should take care of this. I press Ctrl+A, Ctrl+I, Ctrl+S (select all, autoformat, save) pretty much every time I write a line of code. :smile_cat: